### PR TITLE
Update guide.md

### PR DESCRIPTION
**Pull request by drjwbaker**
Per call 7 July, pulled generic info across from DC lesson https://github.com/datacarpentry/OpenRefine-ecology-lesson/blob/gh-pages/_extras/guide.md I'll leave the maintainers to decide what else might go in here!

### CODE REVIEW

File: _extras/guide.md [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
 permalink: /guide/
 ---
-Discussion of the lesson aimed at instructors would normally go here,
-but since this is just an example,
-there isn't much to say.
+
+____
+# Making a handout
+
+Librarians like handouts. To make a handout for this lesson, adapt/print from [http://data-lessons.github.io/library-data-intro/reference/](http://data-lessons.github.io/library-data-intro/reference/).
```
Comment (ccronje) [inline _extras/guide.md]: See what @weaverbel @mkuzak @cmacdonell decide is the best way forward wrt handouts/keypoints versus cheatsheets.

File: _extras/guide.md [hunk 2]
```diff
@@ -1,8 +1,34 @@
...
+* If learners are using a browser other than Firefox, or OpenRefine does not automatically open for them when they click the .exe file, have them point their browser at http://127.0.0.1:3333/ or http://localhost:3333 to launch the program.
+
+* Mac users with the newest operating system will have to allow this to run by "allowing everything" to run. They can change the setting back after the exercise.
+
+* Some students will run into issues with
+  - unzipping
+  - finding the .exe file once the software has been unzipped
+  - finding the data file on their computers after downloading
+  
+ * If OpenRefine crashs when launched from a network share drive, do the following:
```
Comment (ccronje) [inline _extras/guide.md]: Don't quite understand this paragraph.

**Comment by ccronje** [comment]
Thanks @drjwbaker - will wait until @ostephens has a chance to review.

**Comment by ccronje** [comment]
Other comments to consider adding:

From https://github.com/data-lessons/library-openrefine/issues/130:
Tips specific to Refine might relate to the difficulty of getting the text on the screen big enough for live demos and workarounds you've developed. Or that people sometimes download the files as a html page in error.

From meeting http://pad.software-carpentry.org/post-lc2017 (OpenRefine Discussion Agenda):
Contribute our approaches to teaching the 'array' section to the tutor notes. e.g. Owen shows days of week example in a separate project, Tim & Carmi talk through it on whiteboard.

**Comment by ccronje** [comment]
Wrong button sorry

File: _extras/guide.md (hunk 2 above)
Comment (ostephens) [inline _extras/guide.md]: I think the correct terminology would be "Open a Windows Command prompt"
If we need more detail here I think it needs to be slightly more explicit here (i.e. how to open a command prompt, how to change the working directory in Windows)

Comment (weaverbel) [inline _extras/guide.md]: Agree - there is a lot of confusion for windows people between terminal and command prompt. The more explicit the better.

**Comment by ostephens** [comment]
Thanks @drjwbaker and @ccronje - looks good!

**Pull request merged**